Routing/Loader/Reader/RestActionReader.php

```diff
@@ -475,6 +475,17 @@ private function getMethodArguments(\ReflectionMethod $method)
         // ignore all query params
         $params = $this->paramReader->getParamsFromMethod($method);
 
+        // check if a parameter is coming from the request body
+        $ignoreParameters = [];
+        if (class_exists(\Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter::class)) {
```

Comment:
but please also add use case for the classes :)

Reply:
Not sure I understand what you mean, can you elaborate.

Reply:
Oh, I meant `use` statements. 🙈

Reply:
Alright, I prefer that too, but since a lot in the code is with FQN I thought it was some standard or something. Will do that simplification.

Reply:
I changed all FQN in that method so it's cleaned up at the same time.